pages/1.11/security/ent/index.md

```diff
@@ -79,6 +79,10 @@ You can use either of the following methods to determine the security mode of an
 
 - [SSH](/1.11/administering-clusters/sshcluster/) into your master and view the contents of `/opt/mesosphere/etc/bootstrap-config.json`.
 
+- Use the DC/OS UI/Dashboard and go to:
+DC/OS UI -> Cluster -> Overview -> Scroll down to the bottom, to Bootstrap config: Security Mode strict.
+![SECURITY MODE strict](/1.10/img/discover-security-mode.png).
```

Comment:
Thank you, Anitha,

As far as I understand, this is a screenshot of the DC/OS v1.11 and it's only shown on the page which corresponds to DC/OS v1.11. Could you please move it to the correspondent directory: `/1.10/img/`.

Reply:
@adyatlov discover-security-mode.png is present in both 1.11 and 1.10 img folders. 
https://github.com/mesosphere/dcos-docs-site/pull/790

Reply:
I'm sorry @anithamuthu79. I made a typo in my review. What I meant is that we need to move the `/1.10/img/discover-security-mode.png` file to `/1.11/img/discover-security-mode.png` and change the reference in the code from `![SECURITY MODE strict](/1.10/img/discover-security-mode.png).` to `![SECURITY MODE strict](/1.11/img/discover-security-mode.png).`